source/includes/fundamentals/code-snippets/CurrentAPI.java

```diff
@@ -19,23 +19,29 @@ public class CurrentAPI {
 
     public static void main(String[] args) throws InterruptedException {
         CurrentAPI c = new CurrentAPI();
+
+        // Call methods on your CurrentAPI instance that demonstrate MongoDB operations
         c.example1();
         c.example2();
 
     }
 
     private void example1() {
+        // Connect to a MongoDB instance with the current API
```

Comment:
Issue:
Use the indicative mood (verb form that makes a statement). See https://docs.google.com/document/d/1Bnhh1OsGRUuXo_6yVUASGWiSX6sWuqrEFxPP6sLZDHM/edit for examples.
Also, I think "by using" is suitable and more compliant with the [style guide](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#std-term-using).

Suggestion:

Maybe something like the following would be accurate and follow the style guide:
"Connects to MongoDB and retrieves a document by using the non-legacy API"